sources/packages/api/src/route-controllers/education-program-offering/education-program-offering.controller.ts

```diff
@@ -237,7 +246,11 @@ export class EducationProgramOfferingController extends BaseController {
         locationId,
         programId,
         programYearId,
-        selectedIntensity,
+        {
+          offeringIntensity: selectedIntensity,
```

Comment:
Can we rename the _selectedIntensity_ to only _offeringIntensity_ to be aligned with the previous PR changes?
We need also to include the validation of the _OfferingIntensity_ enum.